src/ui/options/components/accounts.tsx

```diff
@@ -36,7 +37,18 @@ const scrobblerPropertiesMap = {
 			placeholder: 'accountsUserTokenPlaceholder',
 		},
 	},
+};
+
+/**
+ * Properties associated with each scrobbler that are stored as an array with items add-and-removable, and the type of input to use to edit them.
+ */
+const scrobblerArrayPropertiesMap = {
 	Webhook: {
+		applicationName: {
+			type: 'text',
```

Comment:
Should we change the `userApiUrl.type` to `url`?

Reply:
Long term I think its very much worth considering (as mentioned I want to completely overhaul the accounts UX in the near future). In the short term I think we should just keep it consistent with the userApiUrl type of the other scrobblers.